tests/osquery/test_osquery_api.py

```diff
@@ -1395,3 +1398,28 @@ def test_continue_file_carving_invalid_block_data(self):
         }
         response = self.post_as_json("carver_continue", post_data)
         self.assertEqual(response.status_code, 400)
+
+    def test_public_urls_are_disabled_on_tests(self):
+        routes = ['enroll', 'config', 'carver_start', 'carver_continue', 'distributed_read', 'distributed_write']
+
+        for route in routes:
+            with self.assertRaises(NoReverseMatch):
+                reverse(f"osquery_public_legacy:{route}")
+            self.assertIsNotNone(reverse(f"osquery_public:{route}"))
+
+    def test_mount_legacy_public_endpoints_flag_is_working(self):
+        url_prefix = "/public"
+        routes = ['enroll', 'config', 'carver_start', 'carver_continue', 'distributed_read', 'distributed_write']
+
+        settings._collection["apps"]._collection["zentral.contrib.osquery"]._collection["mount_legacy_public_endpoints"] = True  # NOQA
```

Comment:
nitpick: maybe set a variable to the app config, to make it shorted, and use it to set it to True and False. It should be possible to avoid the NOQA